src/components/HistoryRecord/HistoryRecord.module.css

```diff
@@ -0,0 +1,24 @@
+.HistoryRecordLongTextChangeButton {
```

Comment:
Is this css file used anywhere?

Reply:
yeap
inside HistoryRecord file

https://github.com/taskany-inc/issues/pull/2588/files#diff-28e7bb86d913c391369eee9c1f6a73620a9b73b57ac0771d78c2eff181931ac4R39